extra/docs_landing.html

```diff
@@ -48,335 +48,178 @@
-          <h4 class="mt-3">
+          <h3 class="mt-3">
             <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt5e0e0ad9a13e6b8c/634d9da18473831f96bbdf1e/security-logo-color-32px.png');"></span>
             Security
-          </h4>
+          </h3>
           <p>Detect, investigate, and respond to threats with Elastic Security for SIEM.</p>
         </div>
       </a>
     </div>
   </div>
-  <div class="row my-4">
-   <div class="col-md-4 col-12 mb-2">
-    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/getting-started-endpoint-security.html">
+<h2 style="font-size:20px;margin-top: 0px;margin-bottom: 0px !important;">Products</h2>
+<div class="row">
+  <div class="col-md-4 col-12 mb-2">
+  <a class="no-text-decoration" href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
+    <div class="card h-100">
+      <h3 class="mt-3">
+        <span class="inline-block float-left icon mr-2" style="background-image: url(
+          'https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt36f2da8d650732a0/5d0823c3d8ff351753cbc99f/logo-elasticsearch-32-color.svg');"></span>
+        Elasticsearch
+      </h3>
+      <p>Elasticsearch is a distributed, JSON-based search and analytics engine.</p>
+    </div>
+  </a>
+  </div>
+  <div class="col-md-4 col-12 mb-2">
+    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/logstash/current/introduction.html">
+      <div class="card h-100">
+        <h3 class="mt-3">
+          <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blta565c71d21f7826f/654bc774b42e39040a69a912/logstash.png');"></span>
+          Logstash
+        </h3>
+        <p>A data collection engine with real-time pipelining capabilities.</p>
+      </div>
+    </a>
+  </div>
+  <div class="col-md-4 col-12 mb-2">
+    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/kibana/current/index.html">
       <div class="card h-100">
-        <h4 class="mt-3">
-          <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt5e0e0ad9a13e6b8c/634d9da18473831f96bbdf1e/security-logo-color-32px.png');"></span>
-          Security
-        </h4>
-        <p>Protect hosts with endpoint threat intelligence from Elastic Security.</p>
+        <h3 class="mt-3">
+          <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt4466841eed0bf232/5d082a5e97f2babb5af907ee/logo-kibana-32-color.svg');"></span>
+          Kibana
+        </h3>
+        <p>Kibana gives shape to your data and is the extensible user interface.</p>
```

Comment:
```suggestion
        <p>Kibana gives shape to your data and is the extensible interface of the Elasticsearch Platform.</p>
```
Sentence seemed unfinished. Removed "user" so it remains on 3 lines.

Reply:
Thanks! Updated in 25786afdf6845856b849844b2ac91bb490c54e40